### DISCUSSION TITLE
Notification when new Comic/Book/Manga is added

### DISCUSSION THREAD

**Discussion by Surgical17** [Kareadita/Kavita#3429] [Ideas]
### Idea Description

A Simple notification feature than can send to Notify or a Webhook to inform of a new comic that was added and detected similar to what Jellyfin or Plex can do.

### Idea Category

Feature Enhancement

### Duration of Using Kavita

3 years

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by scare376** [comment]
Just as a heads up. Joe was considering adding a notification system as part of the Kavita+ subscription.

Also if you want this functionality now. Komf does allow for notifications to a discord webhook/apprise URL.
